Lab1b/Lexic.txt.txt

```diff
@@ -0,0 +1,31 @@
+Alphabet:
+   > uppercase(A-Z) and lowercase(a-z) letters of the English alphabet
+   > underline character '_'
+   > decimal digits (0-9)
+1. Lexic:
+
+a. Special symbols
+- operators: - + * / > < >= <= <-
+- separators: : # ; {} ()
+- reserved keywords: let, read, write, integer, float, string, boolean, True, False, 
+		     forEach, while, cond, else, return
+b. Identifiers
+- <letter>::=A|B|...|Z|a|...|z
+- <digit>::=0|1|...|9
+- <identifier>::= <letter>|<letter><digit>
+
+c. Constants
+1. integer
+   - <zeroNum>::=0
+   - <notZeroNum>::=1|2|...|9
+   - <positiveNum>::=<zeroNum>|<notZeroNum><digit>
+   - <number>::=<positiveNumber>|"-"<positiveNumber>
+2. character
+   - <character>::=<letter>|<digit>
+3. string
+   - <char>::=<letter>|<digit>
```

Comment:
<char> same as <character>

Reply:
Thank you for the notice, appreciate it! I will look into it.